src/pages/Store/StoreDetailPage/index.tsx

```diff
@@ -133,7 +132,10 @@ function StoreDetailPage() {
         </div>
         {storeMenuCategories && storeMenuCategories.length > 0 && (
           <>
-            <div className={styles['menu-title']}>MENU</div>
+            <div className={styles['menu-title__container']}>
+              <div className={styles['menu-title']}>MENU</div>
+              {storeMenus && <UpdateInfo date={storeMenus?.updated_at} />}
```

Comment:
이런식의 조건문이라면 
```suggestion
              {storeMenus?.updated_at && <UpdateInfo date={storeMenus.updated_at} />}
```
이렇게 해줘야 undefined 와 같은 값이 들어가는 걸 방지할 수 있을 것 같습니다.

Reply:
date의 타입을 Date로 고정하고 쓰는쪽에서 파싱하는것도 좋아보이네요!

Reply:
반영하도록 하겠습니다.!!